scripts/compactor_runner.py

```diff
@@ -357,22 +329,118 @@ def _complete_config(self, args):
             config.startCheckpointing = args.startCheckpointing
         return config
 
+    def _configure_server_address(self, compactor_config):
+        """
+        Configure Server hostname and port from args and config file.
+        This method invokes resolve ip method to configure from network interfaces
+        when the hostname is not present in args or config file.
+
+        :param compactor_config: values parsed from config file
+        :return:
+        """
+        # configuring hostname and port
+        pod_name = os.environ.get(POD_NAME, "default_name")
+        pod_namespace = os.environ.get(POD_NAMESPACE, "default_namespace")
+        if pod_name != "default_name" and pod_namespace != "default_namespace":
+            self._config.hostname = pod_name + ".corfu-headless." + pod_namespace + ".svc.cluster.local"
```

Comment:
return statement after this line? 
Or if getting to the port is the requirement, can we configure the port first and then get to hostname configuration?

Reply:
Done, Thank you.